content/modules/redisearch/search-json-quickstart.md

```diff
@@ -0,0 +1,310 @@
+  },
+  "price": 64.99,
+  "stock": 17,
+  "colors": [
+    "black",
+    "white"
+  ]
+}
+```
+
+To store these JSON documents in the database, run the following commands:
+
+```sh
+127.0.0.1:12543> JSON.SET item:1 $ '{"name":"Noise-cancelling Bluetooth headphones","description":"Wireless Bluetooth headphones with noise-cancelling technology","connection":{"wireless":true,"type":"Bluetooth"},"price":99.98,"stock":25,"colors":["black","silver"]}'
+"OK"
+127.0.0.1:12543> JSON.SET item:2 $ '{"name":"Wireless earbuds","description":"Wireless Bluetooth in-ear headphones","connection":{"wireless":true,"type":"Bluetooth"},"price":64.99,"stock":17,"colors":["black","white"]}'
+"OK"
+```
+
+### Search the index
+
+To search the index for JSON documents that match a specified condition, use the [`FT.SEARCH`](https://redis.io/commands/ft.search) command. You can search any field defined in the index schema.
+
+For more information about search queries, see [Search query syntax](https://redis.io/docs/stack/search/reference/query_syntax).
+
+#### Return the entire document
+
+Search query results include entire JSON documents by default.
+
+For example, search for Bluetooth headphones with a price less than 70:
+
+```sh
+127.0.0.1:6379> FT.SEARCH itemIdx '@description:(headphones) @connectionType:(bluetooth) @price:[0 70]'
+1) "1"
+2) "item:2"
+3) 1) "$"
+   2) "{\"name\":\"Wireless earbuds\",\"description\":\"Wireless Bluetooth in-ear headphones\",\"connection\":{\"wireless\":true,\"connection\":\"Bluetooth\"},\"price\":64.99,\"stock\":17,\"colors\":[\"black\",\"white\"]}"
+```
+
+#### Return specific fields
+
+You can use [field projection](https://redis.io/docs/stack/search/indexing_json/#field-projection) to limit the search results to include only specific parts of a JSON document. Use the `RETURN` option to specify which fields to include in the search results.
+
+The following query uses the JSONPath expression `$.stock` to return each item's stock in addition to the name and price:
+
+```sh
+127.0.0.1:6379> FT.SEARCH itemIdx '@description:(headphones)' RETURN 5 name price $.stock AS stock
+1) "2"
+2) "item:1"
+3) 1) "name"
+   2) "Noise-cancelling Bluetooth headphones"
+   3) "price"
+   4) "99.98"
+   5) "stock"
+   6) "25"
+4) "item:2"
+5) 1) "name"
+   2) "Wireless earbuds"
+   3) "price"
+   4) "64.99"
+   5) "stock"
+   6) "17"
+```
+### Aggregate search results
+
+The [`FT.AGGREGATE`](https://redis.io/commands/ft.aggregate) command lets you run a search query and modify the results with operations such as `SORTBY`, `REDUCE`, `LIMIT`, `FILTER`, and more. For a detailed list of available operations and examples, see [Aggregations](https://redis.io/docs/stack/search/reference/aggregations).
+
+You can use [JSON path]({{<relref "/modules/redisjson#redisjson-paths">}}) expressions with the `LOAD` option to run aggregations on JSON documents. You can use any JSON element, even elements that are not included in the index schema.
```

Comment:
Just an editorial note: "you can use" is repeated pretty often. Try to switch up the sentence structure occasionally to avoid repetition. i.e. switch between "use x to do y" and "to do y, use x". For me it usually depends whats more important, the action of doing y, or the method of using x is more important.